_posts/2019-06-14-homebrew-maintainer-meeting.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Homebrew Maintainer Meeting
+author: MikeMcQuaid
+---
+In February 2019 the Homebrew maintainers had our first in-person meeting at and around the [FOSDEM 2019 conference](https://fosdem.org/2019/) in Brussels. Maintainers travelled as far as India and Canada in order to get face-time with each other and have high-bandwidth conversations.
+
+Maintainers arrived on Friday and met informally to socialise and get to know each other. On Saturday we released [Homebrew 2.0.0](https://brew.sh/2019/02/02/homebrew-2.0.0/) and on Saturday and Sunday attended the various talks at the conference (and including a [Homebrew 2.0.0 talk](https://mikemcquaid.com/talks/homebrew-2.0.0/)]).
```

Comment:
I’d remove the “and” from “(**and** including a”.

Reply:
Also, the last `]` is a typo.